packages/react-native-bpk-component-image/src/withLoadingBehaviour.js

```diff
@@ -16,26 +16,27 @@
  * limitations under the License.
  */
 
-import React from 'react';
+/* @flow */
+
+import React, { Component, type ComponentType } from 'react';
 
 // From Recompose: https://github.com/acdlite/recompose/blob/master/src/packages/recompose/getDisplayName.js
-const getDisplayName = Component => {
-  if (typeof Component === 'string') {
-    return Component;
+const getDisplayName = WrappedComponent => {
```

Comment:
We should really move this somewhere else 😛

Reply:
Rule of three :P

Reply:
Good thing Hugo changed the primaryColor HOC to a component then 😄